dotnet/src/SemanticKernel.Skills/Skills.Memory.Qdrant/Skills.Memory.Qdrant.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <RepoRoot>$([System.IO.Path]::GetDirectoryName($([MSBuild]::GetPathOfFileAbove('.gitignore', '$(MSBuildThisFileDirectory)'))))</RepoRoot>
+    </PropertyGroup>
+    <Import Project="$(RepoRoot)/dotnet/nuget/nuget-package.props" />
+
+    <PropertyGroup>
+        <AssemblyName>Microsoft.SemanticKernel.Skills.Memory.Qdrant</AssemblyName>
```

Comment:
I don't think this is a skill, this is a connector, and we should start having a namespace for connectors

Reply:
I think we used to, and then we got rid of it, so there was some confusing on how to namespace this. Definitely agree that it is a connector.

Reply:
@dluc I'm going to resolve this one and we can update w/future connectors.